src/v2/view-builder/shared/polling.js

```diff
@@ -8,14 +8,14 @@ export default {
     const factor = this.options.appState.get('factor');
     const factorPollingInterval = factor && factor.poll && factor.poll.refresh;
     if (_.isNumber(factorPollingInterval)) {
-      this.polling = _.delay(()=>{
-        this.options.trigger('invokeAction', 'factor.poll');
+      this.polling = setInterval(()=>{
```

Comment:
why switch to use setInternal instead of _delay ?

Reply:
because we're no long re-render the view at polling.
probably shall remove the comment.